api/swagger.yml

```
@@ -1114,6 +1114,61 @@ paths:
+              schema:
+                $ref: "#/components/schemas/UpdateGroupMemberGrantsCommand"
+        400:
+          $ref: "#/components/responses/BadRequest"
+        403:
+          $ref: "#/components/responses/Forbidden"
+        404:
+          $ref: "#/components/responses/NotFound"
+        500:
+          $ref: "#/components/responses/Internal"
+      security:
+        - bearerAuth: [ ]
```

YAMLlint (1.37.1)

[error] too many spaces inside empty brackets

(brackets)